docker-bake.hcl

```diff
@@ -0,0 +1,87 @@
+variable "IMAGE" {
+    default = "kubernetes-tools"
+}
+
+variable "TAG" {
+    default = "dev-latest"
+}
+
+variable "CACHE_IMAGE" {
+    default = "sumologic/kubernetes-tools"
+}
+
+variable "BUILD_GO_CACHE_TAG" {
+    default = "go-build-cache"
+}
+
+variable "BUILD_RUST_CACHE_TAG" {
+    default = "rust-build-cache"
+}
+
+variable "TOOLS_CACHE_TAG" {
+    default = "tools-build-cache"
+}
+
+target "multiplatform" {
+    platforms = ["linux/amd64", "linux/arm64"]
+    output = ["type=image"]
+}
+
+target "default" {
+    dockerfile = "Dockerfile"
+    tags = ["${IMAGE}:${TAG}"]
+    cache-from = [
+        "${CACHE_IMAGE}:${BUILD_GO_CACHE_TAG}",
+		"${CACHE_IMAGE}:${BUILD_RUST_CACHE_TAG}",
+		"${CACHE_IMAGE}:${TOOLS_CACHE_TAG}",
```

Comment:
I wonder why that's not indented at the same level (and first entry is at a different level thatn what's on the line 53)

Reply:
Tabs instead of spaces for some reason. I wonder why Github renders it this way, at 4 spaces per tab it should be the same width. It definitely was in VS Code.

Either way, replaces with spaces as it should've been.